.github/workflows/devcontainer.yml

```
@@ -0,0 +1,60 @@
+        with:
+          images: ghcr.io/ktorio/devcontainer
+          tags: |
+            latest
+            {{date 'YYYYMMDD'}}
+
+      - name: Extract Node.js version
+        id: node-version
+        run: echo "result=$(cat .nvmrc)" >> $GITHUB_OUTPUT
+
+      - name: Build image
+        uses: docker/build-push-action@v4
```

actionlint (1.7.7)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)